src/jit/gentree.h

```diff
@@ -2357,9 +2432,10 @@ struct GenTreeCall final : public GenTree
     CORINFO_SIG_INFO* callSig;                // Used by tail calls and to register callsites with the EE
 
     regMaskTP         gtCallRegUsedMask;      // mask of registers used to pass parameters
+
 #ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
-    SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR structDesc;
-#endif // FEATURE_UNIX_AMD64_STRUCT_PASSING
+    MultiRegReturnTypeDesc    gtMultiRegReturnTypeDesc;
+#endif 
```

Comment:
I think it should be unguarded, and should apply to ALL returns. That will make for the cleanest code.